.github/workflows/add-release-reviewers.yml

```
@@ -0,0 +1,29 @@
+name: Add reviewers to release PRs
+
+on:
+  pull_request_target:
+    branches: ['master', 'next']
+  pull_request:
+    types: ['labeled']
```

Shouldn't this `on` clause be more like the following?

```suggestion
  pull_request_target:
    branches: ['master', 'next']
    types: ['labeled']
```

The way I'm reading the current one is that

```
ON pull_request_target IF branches IN [master, next]
OR
ON pull_request IF types IN [labeled]
```

While my solution should replace the `OR` to an `AND`